Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -1129,13 +1156,7 @@ def checkDefault(self):
             if sourceDescriptor.copyInfo:
                 # we choose you!
                 flaggedDefaultCandidate = sourceDescriptor
-        masterLocations = [src.location for src in self.sources]
-        mutatorBias = biasFromLocations(masterLocations, preferOrigin=False)
-        c = [src for src in self.sources if src.location==mutatorBias]
-        if c:
-            mutatorDefaultCandidate = c[0]
-        else:
-            mutatorDefaultCandidate = None
+        mutatorDefaultCandidate = self.getMutatorDefaultCandidate()
```

Comment:
Here I moved into a new method the part of `checkDefault()` that actually needs MutatorMath, with the idea that another python project that uses both `fonttools` and `mutatorMath` can override this method to re-enable the implementation.